src/escalus_story.erl

```diff
@@ -191,9 +191,11 @@ clients_from_resource_counts(Config, ResourceCounts) ->
      || {{_, UserSpec}, ResCount} <- zip_shortest(NamedSpecs,
                                                   ResourceCounts)].
 
-resources_per_spec(UserSpec, ResCount) ->
+resources_per_spec(UserSpec, ResCount) when is_integer(ResCount)->
     [{UserSpec, list_to_binary("res" ++ integer_to_list(N))}
-     || N <- lists:seq(1, ResCount)].
+     || N <- lists:seq(1, ResCount)];
+resources_per_spec(UserSpec, Res) when is_list(Res)->
```

Comment:
Why make the resource a list?
First, the nature of strings and lists in Erlang makes it hard to discern between them with 100% certainty.
Second, username and server are already binaries, so making the resource a binary maintains consistency.